docs/sources/datasources/azuremonitor.md

```diff
@@ -265,6 +265,8 @@ types of template variables.
 | -------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
 | `workspaces()`                                     | Returns a list of workspaces for the default subscription.                                             |
 | `workspaces(12345678-aaaa-bbbb-cccc-123456789aaa)` | Returns a list of workspaces for the specified subscription (the parameter can be quoted or unquoted). |
+| `resources()`                                      | Returns a list of resources for the default subscription.                                              |
+| `resources(12345678-aaaa-bbbb-cccc-123456789aaa)`  | Returns a list of resources for the specified subscription (the parameter can be quoted or unquoted).  |
```

Comment:
```suggestion
| `resources(12345678-aaaa-bbbb-cccc-123456789aaa)`  | Returns a list of resources for the specified subscription (the parameter can be within quotes or it can be without quotes).  |
```